src/main/java/chatBot/task/Task.java

```diff
@@ -0,0 +1,62 @@
+package chatBot.task;
+
+import chatBot.exception.EmptyDescException;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Task can be a ToDoTask, Deadline or an Event.
+ * All tasks have a description and a marker isDone to indicate whether the task has been completed.
+ */
+public abstract class Task {
+    protected static final DateTimeFormatter ORIGINALFORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    protected static final DateTimeFormatter PRINTFORMAT = DateTimeFormatter.ofPattern("MMM d yyyy");
```

Comment:
Should these constants have an underscore separating words (screaming snake case for static final variables)?